Tools/scripts/pathfix.py

```diff
@@ -33,16 +43,18 @@
 new_interpreter = None
 preserve_timestamps = False
 create_backup = True
+add_flag = None
 
 
 def main():
     global new_interpreter
     global preserve_timestamps
     global create_backup
-    usage = ('usage: %s -i /interpreter -p -n file-or-directory ...\n' %
+    global add_flag
```

Comment:
I dislike global variables :-( Maybe all "options" and be stored in a "options" variable (namespace) when updating the code to argparse.